compose-env-values

```diff
@@ -5,9 +5,9 @@ PARITY_KEYFILE1=/ocean-provider/tests/resources/consumer_key_file.json
 PROVIDER_ADDRESS=068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0
 
 NETWORK_URL=wss://rinkeby.infura.io/ws/v3/357f2fe737db4304bd2f7285c5602d0d
-AQUARIUS_URL=https://aquarius.marketplace.dev-ocean.com
```

Comment:
@mariacarmina  do we still need these changes? why?

Reply:
We don't need them I just saw that the URLs were deprecated.